README.rst

```diff
@@ -36,3 +36,5 @@ Come join us on `Freenode <https://freenode.net/>`_! We've got two channels:
 * #bodhi - We use this channel to discuss upstream bodhi development
 * #fedora-apps - We use this channel to discuss Fedora's Bodhi deployment (it is more generally
   about all of Fedora's infrastructure applications.)
+
+Want to know about bodhi `read the documentation <https://bodhi.fedoraproject.org/docs>`_.
```

Comment:
Putting this here makes it appear to be part of the IRC section. All we really need to do is to change the link on line 12 to be this link.